handbook/engineering/web/goals.md

```diff
@@ -4,29 +4,24 @@
 
 ### Long term
 
-**_Deliver the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users._**
+**_The web team has three areas of focus: creating and maintaining a highly usable and intentionally designed webapp interface; delivering the full, unique value of [extensions](https://docs.sourcegraph.com/extensions) to our users; and developing code insights into an entirely new featureset._**
```

Comment:
If we're going to move away from a common theme for the web team and instead list our focus areas explicitly (I'm on board with that) then I think the list would be: the web app, the extensions platform, code host integrations, and code insights. Or the alternative is the way it's described in the [web team page](https://github.com/sourcegraph/about/blob/main/handbook/engineering/web/index.md#growth-plan) where we split it into "web app" and "extensions and integrations".

```suggestion
**_The web team has four areas of focus: creating and maintaining a highly usable and intentionally designed webapp interface; building and growing the [extensions platform](https://docs.sourcegraph.com/extensions); maintaining and expanding code host integrations; and developing code insights into an entirely new featureset._**
```

Reply:
If we look at the [current proposed org structure for 2021](https://about.sourcegraph.com/handbook/engineering/2021_org), it seems the web team could split into 3 distinct teams:
1. Extensibility Team
1. Code Host Integrations Team
1. Web Infrastructure Team

The one area not covered in the org plan is Code Insights.

Considering the org plan has code host integrations and extensibility (which I read as extensions platform) as separate teams I think it makes sense to go with Marek's update of having 4 categories.

Reply:
@jeanduplessis I agree, but note that the org plan has not been updated since it was created 4 months ago. Imo the org plan shouldn't influence our goals, rather our goals should influence the org plan.

Reply:
@felixfbecker thanks for the context and agree that the org plan should follow the goals.

I think Web Infrastructure and Code Insights are easy to rationalize as distinct categories.

The question in my mind then becomes whether native code host integrations (incl the browser extension) should be group as part of the extension platform or as a separate category? 

@felixfbecker do you have any thoughts on this?

Reply:
Asked differently, given an unlimited ability to split the team, would it makes sense to have a team dedicated to the browser extension and native code host integrations, or would it still make sense to rather group it with the extensions platform work?

Reply:
> the org plan shouldn't influence our goals, rather our goals should influence the org plan.

Correct. The next decision point is "when the web team is too big for one manager, how will we split it?"